pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -110,6 +110,10 @@ public void calculateBrokerHostUsage() {
         double elapsedSeconds = (now - lastCollection) / 1000d;
         double cpuUsage = getTotalCpuUsage(elapsedSeconds);
```

Comment:
Nit: it looks like this `getTotalCpuUsage` method updates state for `lastCpuUsage` and `lastCpuTotalTime`. I don't think we want to update those unless we're going to completely update the cpu stats. I think we should move the early return conditional block to before this line 111.

Reply:
OK  I  will  fix

Reply:
I have moved the early return conditional block to before getTotalCpuUsage. PTAL,thanks! @michaeljmarshall